### PR TITLE
BNF-1260 Add interactions introduction page

### PR DESCRIPTION
**Pull request by barkertron**
https://nicedigital.atlassian.net/browse/BNF-1260

### CODE REVIEW

File: gatsby/static/web.config [hunk 1]
```diff
@@ -21,6 +21,7 @@
 					<!-- A to Z list pages are now plural URLs rather than the old, singular style -->
 					<add key="/drug/" value="/drugs/" />
 					<add key="/interaction/" value="/interactions/" />
+					<add key="/interaction/introduction.html" value="/interactions/introduction" />
```
Comment (ediblecode) [inline gatsby/static/web.config:24]: Please can you add a trailing slash?

File: gatsby/src/pages/interactions/introduction.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+			></div>
+		</Layout>
+	);
+};
+
+export const query = graphql`
+	{
+		bnfInteractionsIntroduction {
+			sections {
+				content
```
Comment (ediblecode) [inline gatsby/src/pages/interactions/introduction.tsx:57]: We might be able to use the `...RecordSection` fragment here like we do in other places, assuming the intro is a simple record/record sections

File: gatsby/src/pages/interactions/introduction.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+					Interactions
+				</Breadcrumb>
+				<Breadcrumb>Introduction</Breadcrumb>
+			</Breadcrumbs>
+
+			<PageHeader id="content-start" heading="Interactions: Introduction" />
+
+			<div
+				dangerouslySetInnerHTML={{
+					__html: data.bnfInteractionsIntroduction.sections[0].content,
```
Comment (ediblecode) [inline gatsby/src/pages/interactions/introduction.tsx:46]: I'm not sure we should rely on it being only a single entry in the array. Maybe we can use the `RecordSectionsContent` component here to loop through the list?

File: gatsby/src/pages/interactions/introduction.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+import { PageHeader } from "@nice-digital/nds-page-header";
+
+import { Layout } from "@/components/Layout/Layout";
+import { SEO } from "@/components/SEO/SEO";
+import { useSiteMetadata } from "@/hooks/useSiteMetadata";
+
+export interface InteractionsIntroductionPageProps {
+	data: {
+		bnfInteractionsIntroduction: {
+			sections: { content: string }[];
```
Comment (ediblecode) [inline gatsby/src/pages/interactions/introduction.tsx:14]: We can probably use `RecordSection` here, see {BnfAboutSection.slug}

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: gatsby/src/pages/interactions/introduction.tsx (hunk 3 above)
Comment (barkertron) [reply gatsby/src/pages/interactions/introduction.tsx:46]: Am now looping through the array, but we can't use RecordSectionsContent without a lot of refactoring as the introductions content doesn't have an order property.

File: gatsby/src/pages/interactions/introduction.tsx (hunk 4 above)
Comment (barkertron) [reply gatsby/src/pages/interactions/introduction.tsx]: It differs as it contains an ID property as well as an order property, so I've just specified the fields as part of the new type if that's ok.

**Review by ediblecode — DISMISSED** [review]

File: gatsby/static/web.config [hunk 5]
```diff
@@ -30,6 +31,8 @@
 					<add key="/treatment-summary/bacillus-calmette-guérin-vaccine-2.html" value="/treatment-summaries/bacillus-calmette-guerin-vaccine/" />
 					<add key="/medicinal-forms/bacillus-calmette-guérin.html" value="/drugs/bacillus-calmette-guerin/medicinal-forms/" />
 					<add key="/medicinal-forms/bacillus-calmette-guérin-vaccine.html" value="/drugs/bacillus-calmette-guerin-vaccine/medicinal-forms/" />
+					<!-- Internal links embedded in feed content -->
+					<add key="/guidance/PHP97237" value="/medicines-guidance/adverse-reactions-to-drugs/" />
```
Comment (ediblecode) [inline gatsby/static/web.config:35]: Please can you remove these 2 lines, now that we've added the `@html` directive to the GQL schema?

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: gatsby/static/web.config (hunk 5 above)
Comment (barkertron) [reply gatsby/static/web.config]: Aha, thanks!

**Review by ediblecode — DISMISSED** [review]

**Review by ediblecode — APPROVED** [review]

**Pull request merged**